devel/ansible/roles/dev/files/.bashrc

```diff
@@ -11,6 +11,7 @@ fi
 shopt -s expand_aliases
 alias bdocs="pushd /home/vagrant/bodhi/docs && make html && make man; popd"
 alias blog="sudo journalctl -u bodhi"
+alias btail="sudo journalctl -lfu bodhi"
```

Comment:
You can already achieve this by using ```$ blog -f```, since ```blog``` is just an alias. You can add any other journalctl flags you like with it too. I'd advocate that we make that clearer in the motd rather than making more aliases.